### ISSUE TITLE
Feedback for Service Bus Preview README and samples

### ISSUE THREAD

**Issue by lilyjma** [Azure/azure-sdk-for-python#10693]
I went through all the code samples in the readme and the samples folder. What I found: 

1. `pip install azure-servicebus --pre` installs version 0.50.2 instead of 7.0.0b1
2. send message in readme missing small things:

```
from azure.servicebus import ServiceBusClient //missing Message
import os
connstr = os.environ['SERVICE_BUS_CONN_STR']

with ServiceBusClient.from_connection_string(connstr) as client:
    with client.get_queue_sender(queue_name): //also need “as queue_sender”
        message = Message("Single message")
        queue_sender.send(message)

```
3. There's no doc on how to create a queue (how to enable sessions and other features)
4. client_identity_authentication.py intends to illustrate how `EnvironmentCredential()`, `DefaultAzureCredential() ` can be used to easily authenticate. It might be easier to the user to have this [link](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity) in the sample so they don't have to look to find doc on the identity library, especially for those who aren't familiar with the repo structure.

**Comment by yunhaoling** [comment]
Hi @lilyjma , what's your python version, OS platform?

When we're running `pip install azure-servicebus --pre` locally, it installed the 7.0.0b1 for me

**Comment by lilyjma** [comment]
It was probably because I tried installing the package before it was officially published, so I had to specify 7.0.0b1 to get that version instead of just using --pre. Now --pre is fine.
